### DISCUSSION TITLE
Does It Support MultiProcess?

### DISCUSSION THREAD

**Discussion by jesson3264** [cesanta/mongoose#2820] [Ask questions, get answers]
- My goal is: Support Multi Process
- My actions were: _describe_
- My expectation was: _describe_
- The result I saw: _provide logs or description_
- My question is: _your question_

# Environment

- mongoose version: _X.Y_
- Compiler/IDE and SDK: ZZZ version _X.Y_
- Target hardware/board: _?_
- Connectivity chip/module: _?_
- Target RTOS/OS (if applicable): _?_

**Comment by scaprile** [comment]
Please don't open an issue to ask a question. Moved it to the proper section.
Please see our [documentation](https://mongoose.ws/documentation/),  and follow the guidelines in our [tutorials](https://mongoose.ws/documentation/#tutorials-and-examples).
https://mongoose.ws/documentation/tutorials/core/multi-threaded/

**Comment by jesson3264** [comment]
Thanks for your attention. I'm confused, does it supported  multiprocess,  not  multi threading.

**Comment by scaprile** [comment]
Maybe if you explain what you want to do, my guessing would be better.
